internal/cloud/cloud.go

```diff
@@ -127,6 +115,51 @@ func Shell(ctx context.Context, w io.Writer, projectDir string, githubUsername s
 	return shell(username, vmHostname, projectDir, telemetryShellStartTime)
 }
 
+// Temporary function to create a vm and print vmHostname to be used by devbox extension
```

Comment:
I was assuming this was non-temporary!

> InitVM ensures a vm is started for the user

Reply:
could be non-temporary, depending on how we utilize this for extension implementation. But I was thinking temporary till the gateway can send vm and username information so that I can continue with extension development work